content/en/releases/patch-releases.md

```diff
@@ -78,9 +78,9 @@ releases may also occur in between these.
 
 | Monthly Patch Release | Cherry Pick Deadline | Target date |
 | --------------------- | -------------------- | ----------- |
-| March 2024            | 2024-03-08           | 2024-03-13  |
-| April 2024            | 2024-04-12           | 2024-04-17  |
+| April 2024            | 2024-04-12           | 2024-04-16  |
 | May 2024              | 2024-05-10           | 2024-05-15  |
```

Comment:
Hm, what was the reason to go for a Wednesday here instead of May 14 (Tue) :thinking:

Reply:
I think that was an artifact of the last time this got updated (which is probably why we had to shift the April patches to Tuesday).

We can fix that here and align on Tuesday.

Reply:
@saschagrunert  updated May release to be 2024-05-14 to align w/ Tuesday release day.